### PR TITLE
Removed 'missing' text for optional additional order fields

### PR DESCRIPTION
**Pull request by donaldthai**
## Description

This PR removed the "missing" text for optional additional order fields.

## Code Review Verification Steps

* [ ] End to end tests pass (`make e2e_test`).
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [ ] Any migrations/schema changes also update the diagram in docs/schema/dp3.sqs
* [ ] There are no aXe warnings for UI.
* [ ] This works in IE.
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @willowbl00
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/160327915) for this change

## Screenshots
Basics tab
image

Orders Viewer
image

### CODE REVIEW

**Comment by donaldthai** [comment]
Made some additional changes to the PanelField and replaced the conditional logic mostly in the OrdersViewerPanel.js file.

Please review!

File: src/scenes/Office/AccountingPanel.jsx [hunk 1]
```diff
@@ -22,32 +18,22 @@ const AccountingDisplay = props => {
...
-          />
-        ) : (
-          <PanelField title="Department indicator" className="missing">
-            missing
-          </PanelField>
-        )}
+        <PanelSwaggerField
+          title="Department indicator"
+          fieldName="department_indicator"
+          nullWarning
```
Comment (tinyels) [inline src/scenes/Office/AccountingPanel.jsx]: This is a great improvement! However, I am not sure about the prop name `nullWarning`. Perhaps use `required` to be consistent with `SwaggerField`

Comment (stangah) [reply src/scenes/Office/AccountingPanel.jsx]: I think I came up with that unfortunate name, `required` makes more sense

Comment (donaldthai) [reply src/scenes/Office/AccountingPanel.jsx]: I will change "nullWarning" to "required" for the PanelField component for now. For the PanelSwaggerField, seems like "nullWarning" is used in other areas besides the Office pages.

Comment (donaldthai) [reply src/scenes/Office/AccountingPanel.jsx]: hmmm, changing the prop name to ```required``` is not too bad. I'll do it!

**Comment by donaldthai** [comment]
Hey @tinyels , made all the changes you requested including the jest tests. Please review!

**Pull request merged**